docs/tutorials/_template.ipynb

```diff
@@ -109,19 +109,6 @@
     "[Put all your imports and installs up into a setup section.]"
```

Comment:
<p>Shouldn't this be <code>~=0.7</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/895/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
My bad. Let me change that.